kDriveCore/Data/Models/File.swift

```diff
@@ -60,6 +61,8 @@ public enum ConvertedType: String, CaseIterable {
             return KDriveResourcesAsset.url.image
         case .video:
             return KDriveResourcesAsset.fileVideo.image
+        case .searchExtension:
+            return UIImage(systemName: "magnifyingglass")!
```

Comment:
@valentinperignon I used SF Symbols for a quick hack. Works well, will check if there is something homemade to use

Reply:
We indeed have a search icon: `KDriveResourcesAsset.search.image`

Reply:
ty